.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+
+updates:
+  # Keep dependencies for GitHub Actions up-to-date
+  - package-ecosystem: 'github-actions'
+    directory: '/'
+    schedule:
+      interval: 'daily'
```

Comment:
Can this be weekly instead?  Gives us time to review before we get the next one.

Reply:
Yes the check can be done weekly , But since it is only for Github Actions that usually updates in a month, it won't be as frequent.

Reply:
Good enough then. :)